runtime_lib/aie_api/aieapi.cmake

```diff
@@ -0,0 +1,13 @@
+
+function(add_aie_api_headers SRCPATH BUILDPATH INSTALLPATH)
+  message("Installing aie_api includes from ${SRCPATH} in ${BUILDPATH}")
+
+  # copy header files into build area
+  install(DIRECTORY ${SRCPATH}/ DESTINATION ${BUILDPATH}/aie_api)
```

Comment:
I am curious about all these trailing `/` in the project.

Reply:
Copy the entire contents of the directory in cmake

Reply:
Interesting. You mean that without a `/` it would only create an empty directory?

Reply:
```
A trailing slash will leave the last component empty and install the contents of the input directory to the destination. 
```

https://cmake.org/cmake/help/book/mastering-cmake/chapter/Install.html

Reply:
Interesting. With CMake I learn all my life long!
Actually this is the equivalent of using `/.` in as with normal Unix copy-related commands.